docs/framework/wpf/advanced/opentype-font-features.md

```diff
@@ -191,22 +177,20 @@ Text using OpenType contextual swashes
 ## Alternates  
  Alternates are glyphs that can be substituted for a standard glyph. [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] fonts, such as the Pericles font used in the following examples, can contain alternate glyphs that you can use to create different appearances for text. The following text displays standard glyphs for the Pericles font.  
   
- ![Text using OpenType standard glyphs](./media/opentypefont01.gif "opentypefont01")  
-Text using OpenType standard glyphs  
+ ![Text using OpenType standard glyphs](./media/opentype-font-features/opentype-standard-glyphs.gif "Text using OpenType standard glyphs")  
+  
   
  The Pericles [!INCLUDE[TLA#tla_opentype](../../../../includes/tlasharptla-opentype-md.md)] font contains additional glyphs that provide stylistic alternates to the standard set of glyphs. The following text displays stylistic alternate glyphs.  
   
- ![Text using OpenType stylistic alternate glyphs](./media/opentypefont02.gif "opentypefont02")  
-Text using OpenType stylistic alternate glyphs  
+ ![Text using OpenType stylistic alternate glyphs](./media/opentype-font-features/opentype-stylistic-alternate-glyphs.gif "Text using OpenType stylistic alternate glyphs")  
   
  The following markup example shows how to define stylistic alternate glyphs for the Pericles font, using properties of the <xref:System.Windows.Documents.Typography> object.  
   
  [!code-xaml[OpenTypeFontSamples#2](~/samples/snippets/csharp/VS_Snippets_Wpf/OpenTypeFontSamples/CS/PageOne.xaml#2)]  
   
  The following text displays several other stylistic alternate glyphs for the Pericles font.  
   
- ![Text using OpenType stylistic alternate glyphs](./media/opentypefont03.gif "opentypefont03")  
-Text using OpenType stylistic alternate glyphs  
+ ![Text using OpenType stylistic alternate glyphs](./media/opentype-font-features/opentype-stylistic-alternate-glyphs-periclese.gif "Text using OpenType stylistic alternate glyphs")  
```

Comment:
we should rename the file name from Periclese -> Periclese. Also CATS is complaining about a dupe alt text so let's switch to something else.

```suggestion
 ![Text using OpenType stylistic alternate glyphs  for the Pericles font](./media/opentype-font-features/opentype-stylistic-alternate-glyphs-pericles.gif "Text using OpenType stylistic alternate glyphs  for the Pericles font")
```

Reply:
Updated file name and alt text based on suggestions.